### ISSUE TITLE
Error when trying to migrate from build-in jsdoc rules to es-plugin-jsdoc

### ISSUE THREAD

**Issue by doberkofler** [gajus/eslint-plugin-jsdoc#121]
I have been trying to migrate from the build-in jsdoc rules to the eslint-plugin-jsdoc but receive the following error message when running eslint 5.10.0. I'm sure I'm doing something wrong, but cannot seem to find what.

```
TypeError: Cannot read property 'loc' of null
    at Object.utils.classHasTag (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint-plugin-jsdoc\dist\iterateJsdoc.js:78:63)
    at D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint-plugin-jsdoc\dist\rules\requireParam.js:27:42
    at checkJsdoc (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint-plugin-jsdoc\dist\iterateJsdoc.js:149:7)
    at listeners.(anonymous function).forEach.listener (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\util\safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\util\safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\util\node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\util\node-event-generator.js:280:22)
    at NodeEventGenerator.enterNode (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\util\node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (D:\MyDev\ljs_app\trunk\periscope\build\node_modules\eslint\lib\code-path-analysis\code-path-analyzer.js:632:23)
```

The key parts of my `.eslintrc.js`file are:

```
module.exports = {
	'parser': 'babel-eslint',
	'parserOptions': {
		'ecmaVersion': 2017,
		'sourceType': 'script'
	},
	'plugins': [
		'jsdoc'
	],
	'settings': {
	},
	'env': {
		'es6': true
	},
	'extends': [
		'eslint:recommended'
	],
	'globals': {
	},
	'rules': {
		'jsdoc/check-param-names': 'warn',
		'jsdoc/check-tag-names': 'warn',
		'jsdoc/check-types': 'warn',
		'jsdoc/newline-after-description': 'warn',
		'jsdoc/no-undefined-types': 'warn',
		'jsdoc/require-description': 'warn',
		'jsdoc/require-description-complete-sentence': 'warn',
		'jsdoc/require-example': 'warn',
		'jsdoc/require-hyphen-before-param-description': 'warn',
		'jsdoc/require-param': 'warn',
		'jsdoc/require-param-description': 'warn',
		'jsdoc/require-param-name': 'warn',
		'jsdoc/require-param-type': 'warn',
		'jsdoc/require-returns-description': 'warn',
		'jsdoc/require-returns-type': 'warn',
		'jsdoc/valid-types': 'warn'
	}
};
```

**Comment by gajus** [comment]
Could be related to https://github.com/gajus/eslint-plugin-jsdoc/pull/120?

**Comment by doberkofler** [comment]
Yes, the fix for #120 also solved my problem!
